includes/SiteConfiguration.php

```diff
@@ -82,6 +82,7 @@ class SiteConfiguration
         'username' => 'waca',
         'password' => 'waca'
     ];
+    private string $privacyStatementPath = 'https://accounts.wmflabs.org/privacy';
```

Comment:
No, this is a bad default, and is something I missed when moving from a GitHub path linked directly to rendering in-tool.

I updated my local config to debug, but never updated the config defaults.

In WMCS, this will be an on-disk path, probably something like `/srv/adminstriva/privacy.md`